### PR TITLE
Issue 32630

### PR DESCRIPTION
**Pull request by tmac9494**
## Description
Closes 32630
forces border-box property on banner and removes some unnecessary padding between content and button. Also adds a max-width of 100vw as a safe measure

## Testing done
tested locally, fixed ios issue

### CODE REVIEW

**Comment by k80bowman** [comment]
@tmac9494 I'd like to have some more details and screenshots on this PR, please. Also, it looks like it has some changes that were from your [previous pr](https://github.com/department-of-veterans-affairs/component-library/pull/215). @bkjohnson and I had asked a question on that, but never heard back from you. I'd like to get that PR merged first, so that we're not merging in those changes with this PR.

File: packages/web-components/src/components/va-alert/va-alert.css [hunk 1]
```diff
@@ -18,6 +18,8 @@
   display: flex;
   align-items: flex-start;
   justify-content: center;
+  box-sizing: border-box !important;
```
Comment (k80bowman) [inline packages/web-components/src/components/va-alert/va-alert.css:21]: I'd rather we not use the `!important` declaration here.

**Review by k80bowman — CHANGES_REQUESTED** [review]

**Comment by tmac9494** [comment]
> @tmac9494 I'd like to have some more details and screenshots on this PR, please. Also, it looks like it has some changes that were from your [previous pr](https://github.com/department-of-veterans-affairs/component-library/pull/215). @bkjohnson and I had asked a question on that, but never heard back from you. I'd like to get that PR merged first, so that we're not merging in those changes with this PR.

sorry that was my mistake, I'll fix the bad merge and provide screenshots today

**Comment by pdavies88** [comment]
@tmac9494 are these CSS changes still needed for the `va-alert` component? This PR has been open for awhile without any movement I would like to close it if changes are no longer required. Alternatively feel free to submit a bug ticket and we can prioritize the work for an upcoming sprint: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new/choose

**Comment by pdavies88** [comment]
@tmac9494 I am going to close this PR if you would like to have us make some changes please submit a ticket and we will prioritize. Link for ticket submission: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new/choose

**Pull request closed without merging**